workflows/src/main/resources/migration/accounts-migrations-V1.0.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="R3.Corda" id="accounts-migrations-v1.0">
+
+        <!-- AccountInfo schema -->
+
+        <createTable tableName="accounts">
```

Comment:
I've just spotted that `PersistentAccountInfo` inherits from `PersistentState`, which is also then the ID for this table. Think you'll need the two columns defined by `PersistentState` (`transaction_id` and `output_index`) created in this `createTable` block. You'll also need to create the primary key for the table using those columns.